packages/components/src/checkbox/src/CheckboxGroup.module.css

```diff
@@ -88,11 +88,11 @@
     --list-row-gap: var(--hop-CheckboxGroup-list-bordered-md-row-gap);
 }
 
-.hop-CheckboxGroup--horizontal {
+.hop-CheckboxGroup[data-orientation='horizontal'] {
```

Comment:
we use the other char ", not '

Reply:
ya you're right. I wasn't sure if it applied for css too, but I'll fix it :) thanks.